python/fusion_engine_client/messages/defs.py

```diff
@@ -23,6 +23,9 @@ class SatelliteType(IntEnum):
     SBAS = 8
     IRNSS = 9
 
+    def __str__(self):
```

Comment:
Since this is used in a bunch of places does it make sense to make a common parent class for these?

Reply:
Yes, that's probably a good idea.